.github/CONTRIBUTING.md

```diff
@@ -82,6 +82,13 @@ Let's discuss first!
 
 Let's discuss first!
 
+## Code generation and synchronization
+
+The duckplyr package is a "long-running fork of dplyr", and has code to generate parts of its own implementation and tests, and to synchronize changes from duckplyr with our codebase.
+The main synchronization script is `tools/99-sync.R`, it also contains instructions how to set up a new clone.
+This script should be run whenever the implementation of the verbs changes in a substantial way.
+Watch out for the `# Generated by ...` headers at the top of source files.
```

Comment:
should they use the header that ensure there's a read-only warning in RStudio IDE (and presumedly Positron)?

Reply:
What do you mean by "use the header"?

Reply:
if you open NAMESPACE for instance, `# Generated by roxygen2: do not edit by hand` makes a "Read-only" alert appear.

Reply:
I guess I see what you mean, we can reword in the next iteration.